### ISSUE TITLE
amx_who without IP address, how to fix?

### ISSUE THREAD

**Issue by 2020karaulov2020** [alliedmodders/amxmodx#908]
Latest 1.10 build amx_who without IP address, how to fix?

**Comment by fysiks1** [comment]
amx_who has never had IP addresses so there is nothing to "fix".  The quickest option is to either create or request a custom version of the admincmd plugin with this feature on the AMX Mod X forums [here](https://forums.alliedmods.net/forumdisplay.php?f=12).

**Comment by 2020karaulov2020** [comment]
> amx_who has never had IP addresses so there is nothing to "fix". The quickest option is to either create or request a custom version of the admincmd plugin with this feature on the AMX Mod X forums [here](https://forums.alliedmods.net/forumdisplay.php?f=12).

After update to 1.10 , no IP. In 1.8.# it shows IP.

**Comment by fysiks1** [comment]
No, 1.8.2 doesn't have it.  See this function's official source code in 1.8.2 [here](https://github.com/alliedmodders/amxmodx/blob/3db718799fe42796475e90a7a95df6a3100c53d5/plugins/admincmd.sma#L1008).  If yours had IP then you had a custom version.  I also checked 1.8.0, same thing.
